src/services/positionService.ts

```diff
@@ -629,6 +642,16 @@ export default class PositionService {
     if (yieldFrom || yieldTo) {
       permissions = [...permissions, PERMISSIONS.TERMINATE];
     }
+    const POLYGON_WETH = getWrappedProtocolToken(destinationChainID as number);
+
+    const forwardCallData = getForwardFunctionCallHelper(
+      destinationUSDC as string, // destination USDC
```

Comment:
destinationUSDC seems to be very specific for the connext integration, could we abstract that there?